### PR TITLE
Fix #895 - Resolution/scaling issues on Windows

### LINKED ISSUE #895: A few remaining scaling issues on Windows
I've tried to reproduce this, with the last build (2.202104200913) on Windows 10, using the mentioned scales, and I was not able to do it. The size of the VPN app was changing as expected while going back and forth between different scaling.

But some weird things related to scaling, are still happening. Not sure what did I do :D
scaling issues
 
SysTray icon is having UI issues after few resolution changes.
icon

_Originally posted by @ValentinaPC in https://github.com/mozilla-mobile/mozilla-vpn-client/issues/707#issuecomment-823246883_

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
This fixes a few Windows UI issues when using multiple displays noticed by @bfyee and referenced in #895.

- Fixes UI distortion triggered by "monitor hopping" the app across multiple displays using Window key + arrow left or Windows key + arrow right.
- Prevents the app from being incorrectly resized on Display #1 when changing the scale of Display #2. 

Before:
outputbeforewindows

After:
output400

### CODE REVIEW

File: src/command.cpp [hunk 1]
```diff
@@ -13,6 +13,10 @@
 #include "settingsholder.h"
 #include "simplenetworkmanager.h"
 
+#ifdef MVPN_WINDOWS
+    #include "Windows.h"
```
Comment (lesleyjanenorton) [inline src/command.cpp:17]: Winging it a bit here so please advise! Happy to make changes obviously if there is a better/snazzier/more correct way to do this thing.

Comment (bakulf) [inline src/command.cpp:17]: `<Windows.h>`. In C++, `"` is used mainly for local includes.

**Review by bakulf — APPROVED** [review]

**Comment by bakulf** [comment]
Fix the linter indentation before merging the PR. Thanks!

File: src/command.cpp [hunk 2]
```diff
@@ -115,6 +119,10 @@ int Command::runQmlApp(std::function<int()>&& a_callback) {
   qInstallMessageHandler(LogHandler::messageQTHandler);
   logger.log() << "MozillaVPN" << APP_VERSION;
 
+  #ifdef MVPN_WINDOWS
```
Comment (bakulf) [inline src/command.cpp:122]: remove the 2 spaces before `#idef` end `#endif`

**Review by bakulf — APPROVED** [review]

**Pull request merged**